app/models/designation.js

```diff
@@ -21,7 +22,7 @@ export default Model.extend({
   items:              hasMany('item', { async: true }),
   ordersPackages:     hasMany('ordersPackages', { async: false }),
   orderTransport:     belongsTo('orderTransport', { async: false }),
-  // purposes:           hasMany('purpose', { async: false }),
+  ordersPurposes:     hasMany('orders_purpose', { async: false }),
```

Comment:
Why the Name has inconsistancy?? 
e.g. `belongsTo('orderTransport'` but now we have ` ordersPurposes:     hasMany('orders_purpose'` 
Ideally in JS it should use camelCase `ordersPurpose`
is not it?

Reply:
Yes @shivanibhanwal , Thanks for pointing it out. I'll change it.